.github/workflows/ci_pkg-install.yml

```diff
@@ -27,12 +27,11 @@ jobs:
 
       - name: Prepare env
         run: |
-          pip install check-manifest "twine==3.2" setuptools wheel
+          pip install "twine==3.2" setuptools wheel
 
       - name: Create package
         run: |
-          check-manifest
```

Comment:
why not check the manifest?

Reply:
Because the warnings it outputs doesn't seem to be correct, if you see the changes to `MANIFEST.in`, we just need to include two things (everything else gets included/excluded automatically)

I checked by looking at the generated distribution.

However, `check-manifest` says all the includes/excludes are necessary but doesn't seem like it.
I don't understand why exactly

Reply:
Also, have a look at the PR description